pkg/kubelet/kubelet.go

```diff
@@ -2833,6 +2833,8 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) (*v1.Pod, error) {
 			//TODO(vinaykul,InPlacePodVerticalScaling): Can we recover from this in some way? Investigate
 			klog.ErrorS(err, "SetPodResizeStatus failed", "pod", klog.KObj(allocatedPod))
 		}
+		allocatedPod.Status.Resize = resizeStatus
```

Comment:
is this a bug? or related to the metric?

Reply:
removed